packages/react-native/src/generators/component-story/component-story.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  convertNxGenerator,
+  formatFiles,
+  generateFiles,
+  getProjects,
+  joinPathFragments,
+  normalizePath,
+  Tree,
+} from '@nrwl/devkit';
+import * as ts from 'typescript';
+import {
+  getComponentName,
+  getComponentPropsInterface,
+} from '@nrwl/react/src/utils/ast-utils';
+import { CreateComponentStoriesFileSchema } from './schema';
+
+export function getArgsDefaultValue(property: ts.SyntaxKind): string {
+  const typeNameToDefault: Record<number, any> = {
+    [ts.SyntaxKind.StringKeyword]: "''",
+    [ts.SyntaxKind.NumberKeyword]: 0,
+    [ts.SyntaxKind.BooleanKeyword]: false,
+  };
+
+  const resolvedValue = typeNameToDefault[property];
+  if (typeof resolvedValue === undefined) {
+    return "''";
+  } else {
+    return resolvedValue;
+  }
+}
+
+export function createComponentStoriesFile(
+  host: Tree,
+  { project, componentPath }: CreateComponentStoriesFileSchema
+) {
+  const proj = getProjects(host).get(project);
+  const sourceRoot = proj.sourceRoot;
+
+  // TODO: Remove this entirely, given we don't support TSLint with React?
+  const usesEsLint = true;
```

Comment:
Yup, tslint is dead there's no reason to not use eslint.

Reply:
removed this line